src/components/Lightbox/LightboxGallery.tsx

```diff
@@ -0,0 +1,153 @@
+          .length
+      )
+    );
+  }, []);
+
+  return (
+    <span className="lbx-gal" id={id}>
+      <ModalRB
+        size="xl"
+        show={isOpen}
+        centered
+        animation={false}
+        backdropClassName="mfp-zoom-out-cur" // supposed to add cursor: zoom-out, but broken bc it's behind lightbox div (siblings, lower z-index)
+        onHide={() => setIsOpen(false)}
+        className="mfp-zoom-out-cur"
+      >
+        <ModalRB.Body
+          tabIndex={-1}
+          onKeyDown={(e: { key: string }) => {
+            if (e.key === 'ArrowLeft' || e.key === 'ArrowUp') {
+              setIndex(index === 0 ? numImages - 1 : index - 1);
+            } else if (e.key === 'ArrowRight' || e.key === 'ArrowDown') {
+              setIndex((index + 1) % numImages);
+            }
+          }}
+          bsPrefix="bg-darker"
+          style={{ cursor: 'auto' }}
+        >
+          <button
+            title="Close overlay (escape key)"
+            type="button"
+            className="mfp-close"
+            onClick={() => setIsOpen(false)}
+            style={{ width: 'auto' }}
+          >
+            ×<span className="wb-inv"> Close overlay (escape key)</span>
+          </button>
+          <img
+            className="mfp-img"
+            src={
+              // couldn't assign string|null to src, so I convert it to string, that way null becomes "null" lol
+              images !== undefined && images[index]
+                ? String(
+                    (images[index].childNodes[0] as Element).getAttribute(
+                      'href'
+                    )
+                  )
+                : ''
+            }
+            alt="alternate"
+          />
+          <figcaption>
+            <div className="mfp-bottom-bar" id="lbx-title">
+              <div className="mfp-title">
+                {images !== undefined && images[index]
+                  ? (images[index].childNodes[0] as Element).getAttribute(
+                      'title'
+                    )
+                  : ''}
+              </div>
+              <div className="mfp-counter">
+                Item {index + 1} of {numImages}
+              </div>
+            </div>
+          </figcaption>
+          <button
+            title="Previous (left arrow key)"
+            type="button"
+            className="mfp-arrow mfp-arrow-left mfp-prevent-close"
+            onClick={() => {
+              setIndex(index === 0 ? numImages - 1 : index - 1);
+            }}
+          >
+            <span className="wb-inv"> Previous (left arrow key)</span>
+          </button>
+          <button
+            title="Next (right arrow key)"
+            type="button"
+            className="mfp-arrow mfp-arrow-right mfp-prevent-close"
+            onClick={() => {
+              setIndex((index + 1) % numImages);
+            }}
+          >
+            <span className="wb-inv"> Next (right arrow key)</span>
```

Comment:
Will this text translate if French is needed?

Reply:
No, adding in French version today